target/linux/ramips/dts/mt7628an_qding_qc202.dts

```diff
@@ -0,0 +1,116 @@
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "qding,qc202", "mediatek,mt7628an-soc";
+	model = "Qding QC202";
+
+	leds {
+		compatible = "gpio-leds";
+
+		gpio-relay {

```

Comment:
Hmmmm... relay doesn't count as an LED. Maybe you can just mention it in the commit message instead? There are multiple ways to manipulate GPIOs directly in userspace, you don't have to make it an LED to control it.

Reply:
Ok

Reply:
I think it is better to comment it out instead of deleting it, to give other developers some reference. What do you think?

Reply:
Mentioning it in commit message is enough as a reference. You can also put a human-readable comment at the top of this device tree. Just don't leave it defined as an LED.